stdlib/bool.ml

```diff
@@ -0,0 +1,31 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                                 OCaml                                  *)
+(*                                                                        *)
+(*                         The OCaml programmers                          *)
```

Comment:
I don't know if we care about providing real author names here.  @xavierleroy @damiendoligez : do we care?

What about the copyright owner (is it ok to put INRIA even if no INRIA contributor has been involved in writing the code)?

Reply:
Note that I already made this attribution in the `Result` and `Option` modules. Personally I see little point in having real names on collaboratively edited files and I'm fine with `git blame` for my ego-trips.

Reply:
I don't care personally either, but there might be legal consequences to not mentioning original authors (IANAL).  Hence my request for inputs from the higher authorities.